.github/workflows/veracode-analysis.yml

```diff
@@ -262,8 +262,8 @@ jobs:
           scanpollinginterval: 120  # time between two checks in seconds / [30 to 120]
 
       - name: Promote Scan
-        # only "master" branch will be promoted to policy scan
-        if: success() && github.ref_name == 'master'
+        # only stable branch will be promoted to policy scan
```

Comment:
Hi @Yassir-BenBOUBKER ,
Kindly, could you rebase your branch ?
These changes have been merged last week 🤔 
https://github.com/centreon/drawio/commit/ff6f43afbffe60ecba9a84b77a1f891c53be14ba

Reply:
Also, some conflicts require your attention
`src/main/webapp/js/app.min.js
src/main/webapp/js/integrate.min.js`